build_executable.py

```
@@ -0,0 +1,161 @@
+    # Map system names
+    if system == 'darwin':
+        system = 'macos'
+    elif system == 'windows':
+        system = 'win'
+    
+    return f"{system}-{machine}"
+
+def clean_build_artifacts():
+    """Remove previous build artifacts."""
+    dirs_to_remove = ['build', 'dist', '__pycache__']
+    files_to_remove = ['*.pyc', '*.pyo', '*.spec.bak']
```

Ruff (0.11.9)

Local variable `files_to_remove` is assigned to but never used

Remove assignment to unused variable `files_to_remove`

(F841)